src/vanilla/server.rs

```diff
@@ -104,3 +140,6 @@ impl Server {
         Ok(())
     }
 }
+
+unsafe impl Send for Server {}
+unsafe impl Sync for Server {}
```

Comment:
Now that sfackler/rust-openssl@89e88727ffab25c49e67415ebc6bca555f6358c6 is in this isn't necessary anymore.